gateway/mw_redis_cache.go

```diff
@@ -147,6 +147,9 @@ type cacheOptions struct {
 
 // ProcessRequest will run any checks on the request on the way through the system, return an error to have the chain fail
 func (m *RedisCacheMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Request, _ interface{}) (error, int) {
+	t1 := time.Now()
+	ms := DurationToMillisecond(time.Since(t1))
```

Comment:
since nothing is being done in between `time.Now()` and `DurationToMillisecond(time.Since(t1))` `ms` would be always near to 0 right? is it intended? or am I missing something?

Reply:
Correct, should move to L250 approx, before latency hit. Thanks for catching it